### PR TITLE
Add server commands

### PR DESCRIPTION
**Pull request by NateConroy1**
jet server add [args]
jet server list
jet server --remove <alias>
jet server clear [args]

### CODE REVIEW

File: src/main.cpp [hunk 1]
```diff
@@ -1,38 +1,150 @@
...
 #include "CLI/App.hpp"
-#include "CLI/Formatter.hpp"
 #include "CLI/Config.hpp"
+#include "CLI/Formatter.hpp"
+#include "services/service_manager.h"
 #include "src/version.h"
+#include <algorithm>
+#include <fmt/format.h>
+
+void printServerList(const std::vector<record_server_t>& servers) {
```
Comment (NateConroy1) [inline src/main.cpp:9]: This function is kind of a mess. This is just an attempt to align the list nicely

File: src/main.cpp [hunk 2]
```diff
@@ -1,38 +1,150 @@
...
+    serverAddCommand->add_option("--host", serverHost, "Specify the url of the server instance.")
+        ->required();
+    serverAddCommand->add_option("--username", serverUsername, "Specify the username.")->required();
+    serverAddCommand->add_option("--password", serverPassword, "Specify the password.")->required();
+
+    auto serverListCommand
+        = serverCommand->add_subcommand("list", "List the available server connections.")
+              ->group("Outputs");
+    auto serverRemoveCommand
+        = serverCommand->add_option("--remove", serverAlias, "Remove a server connection.")
```
Comment (NateConroy1) [inline src/main.cpp:70]: I don't think you can add an option with a text argument unless you use the --. Alternatively I could change this to:
`jet server remove --alias <alias>`

**Review by NateConroy1 — COMMENTED** [review]
Right now just storing the passwords as plain text, so we'll have to come back and fix this depending on how we decide to handle hashing.

File: src/services/service_manager.cpp [hunk 3]
```diff
@@ -36,19 +34,10 @@ void ServiceManager::init() {
         setLogLevelViaEnv = true;
     }
 
-    logger->header("  _ _  ______  __     __");
-    logger->header("  __  /_  __/ / /__  / /_");
-    logger->header("   ___ / /_  / / _ \\/ __/");
-    logger->header("  __  / / /_/ /  __/ /_");
-    logger->header(" _ _ /_/\\____/\\___/\\__/");
```
Comment (NateConroy1) [inline src/services/service_manager.cpp:43]: Forgot I removed this, I can add it back if you want to save it. I hardcoded logging to be off for now so it won't display if I leave it.

**Review by chrisdalke — APPROVED** [review]

**Pull request merged**